nixos/modules/programs/droidcam.nix

```
@@ -0,0 +1,16 @@
+{ lib, pkgs, config, ... }:
+
+with lib;
+
+{
+  options.services.droidcam = {
+    enable = mkEnableOption "DroidCam client";
+  };
+
+  config = lib.mkIf cfg.enable {
```

```suggestion
  config = lib.mkIf config.programs.droidcam.enable {
```
`cfg` is not defined.